decentraland/unity-renderer
# feat: update of teleport HUD animations and UI + unify to single teleport prompt

## What does this PR change?

Fix #4452
Adds the new teleport prompt UI and refactors teleport panel to be the same one in both coordinates clicking and scene teleport request UI

## How to test the changes?



1. Launch the explorer https://play.decentraland.org/?explorer-branch=feat/teleport-prompt-animation
2. Type different coordinates in chat
3. Verify that clicking on them opens the new teleport UI
4. Find a scene with a teleport item in the world (like the ice poker scenes)
5. Click on the teleport target
6. Verify that the same teleport popup appears
7. Check that cancel and jump in buttons work as expected

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
